plugins/arm/semantics/aarch64-pstate.lisp

```diff
@@ -0,0 +1,69 @@
+(declare (context (target arm armv8-a+le)))
+
+(in-package aarch64)
+
+;;; SPECIFIC PROCESSOR STATE INSTRUCTIONS
+
+(defmacro CCMN** (rn rm-or-imm nzcv cnd)
+  "(CCMN** rn rm-or-imm nzcv cnd) implements either CCMN*i
+   or CCMN*r. The semantics are the same at the lisp level;
+   an immediate or register value is given as second argument."
+  (if (condition-holds cnd)
+    (set-nzcv-from-registers (+ rn rm-or-imm) rn rm-or-imm)
+    (set-nzcv nzcv)))
+
+(defun CCMNWi (rn imm nzcv cnd) (CCMN** rn imm nzcv cnd))
+(defun CCMNXi (rn imm nzcv cnd) (CCMN** rn imm nzcv cnd))
+(defun CCMNWr (rn rm  nzcv cnd) (CCMN** rn rm  nzcv cnd))
+(defun CCMNXr (rn rm  nzcv cnd) (CCMN** rn rm  nzcv cnd))
+
+(defmacro CCMP** (rn rm-or-imm nzcv cnd)
+  "(CCMP** rn rm-or-imm nzcv cnd) implements either CCMP*i
+   or CCMP*r. The semantics are the same at the lisp level;
+   an immediate or register value is given as second argument."
+  (if (condition-holds cnd)
+    (let ((rm-or-imm (lnot rm-or-imm)))
+      (set-nzcv-from-registers (+ rn rm-or-imm 1) rn rm-or-imm))
+    (set-nzcv nzcv)))
+
+(defun CCMPWi (rn imm nzcv cnd) (CCMP** rn imm nzcv cnd))
+(defun CCMPXi (rn imm nzcv cnd) (CCMP** rn imm nzcv cnd))
+(defun CCMPWr (rn rm  nzcv cnd) (CCMP** rn rm  nzcv cnd))
+(defun CCMPXr (rn rm  nzcv cnd) (CCMP** rn rm  nzcv cnd))
+
+;; CFINV gets turned into MSR (register) by LLVM even though they're not aliases.
+;; --show-knowledge shows that LLVM returned "MSR 0x200 XZR" for CFINV, but
+;; this isn't even a valid opcode? For MSR (register) both arguments need to be registers..
+
+;; (defun CFINV () (set CF (lnot CF)))
```

Comment:
after #1461 `CFINV` will be lifted correctly and the signature matches, so you can uncomment this.